style.css

```diff
@@ -90,3 +99,10 @@ article img:hover {
 .button:hover {
   color: grey;
 }
+
+@media (max-width: 400px) {
```

Comment:
In order to be mobile-first, you should be using min-width and making style changes as the screen size goes from small to large.